docs/customize/authentication.md

```diff
@@ -79,6 +79,29 @@ You can also test ORCID login using the ORCID sandbox environment.
 See the plugin [documentation](https://invenio-oauthclient.readthedocs.io/en/latest/usage.html#module-invenio_oauthclient.contrib.orcid)
 for more information.
 
+#### OpenAIRE
```

Comment:
Is it OpenAIRE or EOSC?

Reply:
> Is it OpenAIRE or EOSC?

It is OpenAIRE-specific... EOSC AAI is more like a collection of "recommendations" that potential services should follow...

(Section moved under Keycloak)